sds/src/proximity_keywords/excluded_keywords.rs

```diff
@@ -0,0 +1,22 @@
+use metrics::Counter;
+
+pub struct CompiledExcludedProximityKeywords {
+    pub look_ahead_character_count: usize,
+    pub keywords_pattern: super::ProximityKeywordsRegex,
```

Comment:
I don't really understand why the `super::` part is needed here. What is the advantage of doing that compared to importing the struct? 🙏

Reply:
It's not needed. Probably just a side-effect of the IDE importing things for me, and lots of refactoring while writing this. I'll remove it.